introduction.md

```diff
@@ -1,2 +1,6 @@
 This is introduction
 JavaScript is my favorite Code School path
+
+Deadly skills:
+* Git
+* JavaScript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?